examples/js/loaders/gltf/glTFLoader.js

```diff
@@ -826,6 +826,8 @@ THREE.glTFLoader.prototype.load = function( url, callback ) {
 									var opacity =  USE_A_ONE ? value : (1.0 - value);
 									uvalue = opacity;
 									params.transparent = true;
+								} else if (value) {
```

Comment:
What are you checking for here?  `value !== undefined`? `value !== false`? `value !== 0`? `value !== null`?